2009-10-29  Ulrich Drepper  <drepper@redhat.com>
 
	[BZ #10391]
	* resolv/res_send.c (send_dg): Initialize resplen.

diff --git a/resolv/res_send.c b/resolv/res_send.c
index fa48bd9..e2bbfcc 100644
--- a/resolv/res_send.c
+++ b/resolv/res_send.c
@@ -1005,7 +1005,8 @@ send_dg(res_state statp,
 	struct pollfd pfd[1];
 	int ptimeout;
 	struct sockaddr_in6 from;
-	int resplen, n;
+	int resplen = 0;
+	int n;
 
 	/*
 	 * Compute time for the total operation.
